DependencyInjection/Configuration.php

```diff
@@ -227,6 +227,17 @@ public function getConfigTreeBuilder()
                 ->end()
             ->end()
 
+            ->arrayNode('templates_admin')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->scalarNode('list')->defaultValue('SonataPageBundle:PageAdmin:list.html.twig')->cannotBeEmpty()->end()
+                    ->scalarNode('tree')->defaultValue('SonataPageBundle:PageAdmin:tree.html.twig')->cannotBeEmpty()->end()
+                    ->scalarNode('compose')->defaultValue('SonataPageBundle:PageAdmin:compose.html.twig')->cannotBeEmpty()->end()
+                    ->scalarNode('compose_container_show')->defaultValue('SonataPageBundle:PageAdmin:compose_container_show.html.twig')->cannotBeEmpty()->end()
+                    ->scalarNode('select_site')->defaultValue('SonataPageBundle:PageAdmin:select_site.html.twig')->cannotBeEmpty()->end()
```

Comment:
Also, vim can use phpcs to validate a file, so I think PHPStorm can do that too.